krcal/map_builder/map_builder_functions_test.py

```diff
@@ -89,6 +91,50 @@ def test_maps_nans_outside_rmax(xy_pos, output_maps_tmdir):
     assert all(np.isnan   (coefs[maskout]))
     assert all(np.isfinite(coefs[maskin ]))
 
+@mark.dependency(depends="test_scrip_runs_and_produces_correct_outputs")
+def test_correct_map_with_unsorted_dst(folder_test_dst  ,
+                                       test_dst_file    ,
+                                       output_maps_tmdir):
+    """
+    This test shuffles the input dst, and checks that the map is the same
+    as the one created with the same sorted dst.
+    """
+    map_file_sort   = os.path.join(output_maps_tmdir, 'test_out_map.h5')
+    map_file_unsort = os.path.join(output_maps_tmdir, 'test_out_unsort.h5')
+    histo_file_out = os.path.join(output_maps_tmdir, 'test_out_histo.h5')
+
+    dst = load_dst(folder_test_dst+test_dst_file, 'DST', 'Events')
+    if "index" in dst:del dst["index"]
+    dst = dst.sort_values(by=['S2e'])
+    tmp_unsorted_dst = 'unsorted_dst.h5'
+    dst.to_hdf(output_maps_tmdir+tmp_unsorted_dst,
+               key     = "DST"  , mode         = "w",
+               format  = "table", data_columns = True,
+               complib = "zlib" , complevel    = 4)
+    with tb.open_file(output_maps_tmdir+tmp_unsorted_dst, "r+") as file:
+        file.rename_node(file.root.DST.table, "Events")
+        file.root.DST.Events.title = "Events"
+
+    default_n_bins = 15
+    run_number     = 7517
+    config = configure('maps $ICARO/krcal/map_builder/config_LBphys.conf'.split())
+    map_params_new = copy.copy(config.as_namespace.map_params)
```

Comment:
why copy?

Reply:
It was a mistake, i'm going to change it.